app/components/obs/inputs/ObsBoolInput.vue.ts

```diff
@@ -1,13 +1,27 @@
 import { Component, Prop } from 'vue-property-decorator';
 import { ObsInput, IObsInput, TObsType } from './ObsInput';
+import VFormGroup from '../../shared/inputs/VFormGroup.vue';
+import { metadata } from '../../shared/inputs';
+import { BoolInput } from '../../shared/inputs/inputs';
 
-@Component
+@Component({
+  components: { VFormGroup, BoolInput },
+})
 class ObsBoolInput extends ObsInput<IObsInput<boolean>> {
   static obsType: TObsType;
 
   @Prop()
   value: IObsInput<boolean>;
 
+  get metadata() {
+    return metadata.bool({
+      // Here we need to hide the description only if showDescription is explicitly false
+      // tslint:disable-next-line:no-boolean-literal-compare
```

Comment:
This tslint rule is fairly dumb.  @blackxored is this designed to catch potential bugs or is this just meant for stylistic purposes?  Honestly I would be in favor of disabling this rule if there isn't a good bug-catching argument for it.

Reply:
We are disabling this rule.